compute/rn/release-information/release-notes-22-01-security-update1.adoc

```diff
@@ -0,0 +1,33 @@
+== 22.01 Security Update 1 Release Notes
+
+The following table outlines the release particulars:
+
+[cols="1,4"]
+|===
+|Build
+|21.08.840
+
+|Code name
+|Joule, 22.01 security update 1
+
+|Release date
+|January 25, 2022
+
+|Type
+|Security Update? Maintenance release?
```

Comment:
Hey @iansk , can you please confirm if this should be Security Update or Maintenance Release, and revise it here?

Reply:
@Pubs-MV This is a re-release of 22.01 (the major release). I've updated accordingly.